unittests/producer_schedule_if_tests.cpp

```diff
@@ -174,7 +174,7 @@ BOOST_FIXTURE_TEST_CASE( proposer_policy_progression_test, legacy_validating_tes
                                };
 
    // start a round of production
-   produce_blocks(config::producer_repetitions-1);
+   produce_blocks(config::producer_repetitions-1, true);
```

Comment:
It seems clearer using a different name for possible skipping voting than passing a `true` argument.

Reply:
If you think it would be better to have a separate argument I can add one, just don't see the point. There are not very many tests that need the functionality so seemed best to roll it into the existing empty arg.

Reply:
No need to change.